test/luatest_helpers/misc.lua

```diff
@@ -21,8 +26,15 @@ local function skip_if_enterprise()
     luatest.skip_if(is_enterprise_package(), 'package is Enterprise')
 end
 
+-- Skips a running test if Tarantool build target is macOS.
+local function skip_if_macOS()
+    luatest.skip_if(is_macOS_build(), 'build target is not Linux')
```

Comment:
Nit: mixing CamelCase and underscores in one name looks ugly. Better name `is_mac_os_build` IMO.

Reply:
Removed this helper in favor of testing for `readline` availability.